tools/ontoversion

```diff
@@ -8,21 +8,11 @@ import argparse
 import sys
 
 import rdflib
+from rdflib.exceptions import ParserError
```

Comment:
Not needed any more as far as I can see
```suggestion
```

Reply:
Yes, you are right. This is not working properly yet though so it is still only a draft pull request, and I am keeping some things in the code for help until when I can look at it again.  I'll let you know when review is needed again.